optuna/visualization/_pareto_front.py

```diff
@@ -140,6 +144,12 @@ def objective(trial):
             ),
         ]
     else:
+        warnings.warn(
+            "``constraint_func`` argument is an experimental feature."
+            " The interface can change in the future.",
+            ExperimentalWarning,
```

Comment:
Should we check this warning in unit tests?
The `multivariate` of `TPESampler` has such a test case.

https://github.com/optuna/optuna/blob/07a8193c34e5e389770c6ae549001bb574510e7a/tests/samplers_tests/tpe_tests/test_sampler.py#L35-L37

Reply:
I added such a test in `test_pareto_front.py`.

Reply:
Thanks!